### PR TITLE
[test] -  Add CSS

### PR DESCRIPTION
**Pull request by Tomo-Furu**
Additions:
- Added style.css, custom-font.css, S6u-w4BMUTPHjxsIPx-mPCLQ7A.woff2 file
- updated app.blade.php related to css

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -10,17 +10,22 @@
     <title>{{ config('app.name', 'Laravel') }}</title>
 
     <!-- Fonts -->
-    <link rel="dns-prefetch" href="//fonts.bunny.net">
-    <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
+    {{-- <link rel="dns-prefetch" href="//fonts.bunny.net"> --}}
+    {{-- <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet"> --}}
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
```
Comment (keanbillyfernandez) [inline resources/views/layouts/app.blade.php]: Better to download this font because in cdn's sometimes it cause some errors

**Comment by keanbillyfernandez** [comment]
May I know where does this files came from?
image

**Comment by Tomo-Furu** [comment]
> May I know where does this files came from? 

First, I copied google api's URL and pasted in a browser.
Codes appeared in a browser, so I copied src:url in the codes and pasted src:url in a browser .
After that, I could download these files.

スクリーンショット 2024-04-13 151557

**Comment by keanbillyfernandez** [comment]
> > May I know where does this files came from?
> 
> First, I copied google api's URL and pasted in a browser. Codes appeared in a browser, so I copied src:url in the codes and pasted src:url in a browser . After that, I could download these files.
> 
> スクリーンショット 2024-04-13 151557

No need for that one, We just need the custom-font.css

**Comment by Tomo-Furu** [comment]
> > > May I know where does this files came from?
> > 
> > 
> > First, I copied google api's URL and pasted in a browser. Codes appeared in a browser, so I copied src:url in the codes and pasted src:url in a browser . After that, I could download these files.
> > スクリーンショット 2024-04-13 151557
> 
> No need for that one, We just need the custom-font.css

Actually, Carlo-san taught me the way using download instead of using URL.
Which should we choose in this situation?

**Comment by keanbillyfernandez** [comment]
> > > > May I know where does this files came from?
> > > 
> > > 
> > > First, I copied google api's URL and pasted in a browser. Codes appeared in a browser, so I copied src:url in the codes and pasted src:url in a browser . After that, I could download these files.
> > > スクリーンショット 2024-04-13 151557
> > 
> > 
> > No need for that one, We just need the custom-font.css
> 
> Actually, Carlo-san taught me the way using download instead of using URL. Which should we choose in this situation?

Okay okay, I saw the code again and it is linked in the custom-font.css

**Pull request merged**